pinot-core/src/main/java/org/apache/pinot/core/auth/BasicAuthUtils.java

```diff
@@ -64,21 +61,22 @@ private BasicAuthUtils() {
    * @return list of BasicAuthPrincipals
    */
   public static List<BasicAuthPrincipal> extractBasicAuthPrincipals(PinotConfiguration configuration, String prefix) {
-      String principalNames = configuration.getProperty(prefix);
-      Preconditions.checkArgument(StringUtils.isNotBlank(principalNames), "must provide principals");
+    String principalNames = configuration.getProperty(prefix);
```

Comment:
does it make sense to also refactor the rest of the utils into common? putting them in core seems weird to me: is there any situation when the rest of the function util is not utilized together with the common one?